ports/stm32/powerctrlboot.c

```diff
@@ -388,25 +390,6 @@ void SystemClock_Config(void) {
     SystemCoreClockUpdate();
     powerctrl_config_systick();
 
-    #if MICROPY_HW_ENABLE_USB
```

Comment:
Why did you remove this code?  It should work with the clock recovery system and HSI.

Reply:
Because STM32L152 does not have clock recovery system and HSI48.
I noticed that `__HAL_RCC_SYSCFG_CLK_ENABLE()` is equivalent to `RCC->APB2ENR |= RCC_APB2ENR_SYSCFGEN` and it is needed only if USB is enabled.
So I modified it (a2c0201)

Reply:
Ah, yes, you are correct, the L1 doesn't have these things.